Building on mac with BUILD_SHARED_LIBS=ON fails

Building on both master and stable-1.1 fail when running

    OPENSSL_ROOT_DIR=/usr/local/opt/openssl cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG \
      -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG \
      -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/freerdp/1.1.0-beta1_1 \
      -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST \
      -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DWITH_X11=ON -DBUILD_SHARED_LIBS=ON
    make

on a mac (I'm currently running on a intel mac with OSX El Capitan 10.11.6).

The failure is at the end while linking the Mac library + client:

```
[100%] Linking C shared library MacFreeRDP.framework/MacFreeRDP
cd /Users/untoldone/Source/FreeRDP/client/Mac && mkdir -p /Users/untoldone/Source/FreeRDP/client/Mac//MacFreeRDP.framework/Versions/1.1.0/Resources
cd /Users/untoldone/Source/FreeRDP/client/Mac && /usr/local/Cellar/cmake/3.6.1/bin/cmake -E cmake_link_script CMakeFiles/MacFreeRDP-library.dir/link.txt --verbose=1
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc   -mssse3 -DWINPR_EXPORTS -DFREERDP_EXPORTS -mmacosx-version-min=10.5 -fobjc-nonfragile-abi -DNDEBUG -dynamiclib -Wl,-headerpad_max_install_names  -o MacFreeRDP.framework/Versions/1.1.0/MacFreeRDP -install_name @executable_path/../../Frameworks/MacFreeRDP.framework/Versions/1.1.0/MacFreeRDP CMakeFiles/MacFreeRDP-library.dir/__/common/client.c.o CMakeFiles/MacFreeRDP-library.dir/MRDPCursor.m.o CMakeFiles/MacFreeRDP-library.dir/MRDPView.m.o CMakeFiles/MacFreeRDP-library.dir/PasswordDialog.m.o CMakeFiles/MacFreeRDP-library.dir/main.m.o CMakeFiles/MacFreeRDP-library.dir/mf_client.m.o -framework Cocoa -framework Foundation -framework AppKit ../common/libfreerdp-client.1.1.0.dylib ../../libfreerdp/gdi/libfreerdp-gdi.1.1.0.dylib ../../libfreerdp/rail/libfreerdp-rail.1.1.0.dylib ../../libfreerdp/cache/libfreerdp-cache.1.1.0.dylib ../../libfreerdp/core/libfreerdp-core.1.1.0.dylib ../../libfreerdp/crypto/libfreerdp-crypto.1.1.0.dylib ../../libfreerdp/locale/libfreerdp-locale.1.1.0.dylib ../../winpr/libwinpr/input/libwinpr-input.0.1.0.dylib /usr/X11R6/lib/libX11.dylib ../../libfreerdp/common/libfreerdp-common.1.1.0-beta1.dylib ../../winpr/libwinpr/library/libwinpr-library.0.1.0.dylib ../../winpr/libwinpr/file/libwinpr-file.0.1.0.dylib ../../winpr/libwinpr/dsparse/libwinpr-dsparse.0.1.0.dylib ../../winpr/libwinpr/sspi/libwinpr-sspi.0.1.0.dylib ../../winpr/libwinpr/crypto/libwinpr-crypto.0.1.0.dylib ../../winpr/libwinpr/rpc/libwinpr-rpc.0.1.0.dylib ../../libfreerdp/codec/libfreerdp-codec.1.1.0.dylib ../../libfreerdp/primitives/libfreerdp-primitives.1.1.0.dylib ../../winpr/libwinpr/registry/libwinpr-registry.0.1.0.dylib ../../winpr/libwinpr/pool/libwinpr-pool.0.1.0.dylib ../../libfreerdp/utils/libfreerdp-utils.1.1.0.dylib ../../winpr/libwinpr/utils/libwinpr-utils.0.1.0.dylib /usr/lib/libz.dylib /usr/lib/libssl.dylib /usr/lib/libcrypto.dylib ../../winpr/libwinpr/sysinfo/libwinpr-sysinfo.0.1.0.dylib -lm ../../winpr/libwinpr/synch/libwinpr-synch.0.1.0.dylib ../../winpr/libwinpr/interlocked/libwinpr-interlocked.0.1.0.dylib ../../winpr/libwinpr/thread/libwinpr-thread.0.1.0.dylib ../../winpr/libwinpr/handle/libwinpr-handle.0.1.0.dylib ../../winpr/libwinpr/path/libwinpr-path.0.1.0.dylib ../../winpr/libwinpr/crt/libwinpr-crt.0.1.0.dylib ../../winpr/libwinpr/heap/libwinpr-heap.0.1.0.dylib ../../winpr/libwinpr/environment/libwinpr-environment.0.1.0.dylib -Wl,-rpath,"\$ORIGIN/../lib:\$ORIGIN/.."
freerdp-client
cd /Users/untoldone/Source/FreeRDP/client/Mac && /usr/local/Cellar/cmake/3.6.1/bin/cmake -E copy /Users/untoldone/Source/FreeRDP/client/common/libfreerdp-client.1.1.0.dylib /Users/untoldone/Source/FreeRDP/client/Mac//MacFreeRDP.framework/Contents/libfreerdp-client.1.1.0.dylib
cd /Users/untoldone/Source/FreeRDP/client/Mac && install_name_tool -change /Users/untoldone/Source/FreeRDP/client/common/libfreerdp-client.1.1.dylib @executable_path/../Frameworks/MacFreeRDP.framework/Contents/libfreerdp-client.1.1.0.dylib /Users/untoldone/Source/FreeRDP/client/Mac//MacFreeRDP.framework/MacFreeRDP-library
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/install_name_tool: can't open file: /Users/untoldone/Source/FreeRDP/client/Mac//MacFreeRDP.framework/MacFreeRDP-library (No such file or directory)
make[2]: *** [client/Mac/MacFreeRDP.framework/Versions/1.1.0/MacFreeRDP] Error 1
make[2]: *** Deleting file `client/Mac/MacFreeRDP.framework/Versions/1.1.0/MacFreeRDP'
make[1]: *** [client/Mac/CMakeFiles/MacFreeRDP-library.dir/all] Error 2
make: *** [all] Error 2
```

--- a/client/Mac/CMakeLists.txt
+++ b/client/Mac/CMakeLists.txt
@@ -117,11 +117,11 @@ if (${BUILD_SHARED_LIBS})
 			COMMENT "Copying ${LIB} to output directory"
 			COMMAND install_name_tool -change "$<TARGET_SONAME_FILE:${LIB}>"
 				"@executable_path/../Frameworks/${MODULE_OUTPUT_NAME}.framework/Contents/$<TARGET_FILE_NAME:${LIB}>"
-				"${CMAKE_CURRENT_BINARY_DIR}/$(CONFIGURATION)/${MODULE_OUTPUT_NAME}.framework/${MODULE_NAME}"
+				"${CMAKE_CURRENT_BINARY_DIR}/$(CONFIGURATION)/${MODULE_OUTPUT_NAME}.framework/${MODULE_OUTPUT_NAME}"
 			COMMENT Setting install name for ${LIB}
 			COMMAND "${CMAKE_COMMAND}" -E echo install_name_tool -change "$<TARGET_SONAME_FILE:${LIB}>"
 				"@executable_path/../Frameworks/${MODULE_OUTPUT_NAME}.framework/Contents/$<TARGET_FILE_NAME:${LIB}>"
-				"${CMAKE_CURRENT_BINARY_DIR}/$(CONFIGURATION)/${MODULE_OUTPUT_NAME}.framework/${MODULE_NAME}")
+				"${CMAKE_CURRENT_BINARY_DIR}/$(CONFIGURATION)/${MODULE_OUTPUT_NAME}.framework/${MODULE_OUTPUT_NAME}")
 	endforeach()
 
 	# Call install_name_tool to reassign the library install names in dependent libraries

